### PR TITLE
Update Switcher docs for button platform

### PR DESCRIPTION
**Pull request by thecode**
## Proposed change

Update docs for Switcher button platform

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/81245
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/switcher_kis.markdown [hunk 1]
```diff
@@ -37,17 +37,30 @@ If you completed the integration setup but are still unable to control the devic
 
 {% include integrations/config_flow.md %}
 
+## Buttons
+
+For Switcher Breeze the integration provides the following buttons:
+
+- For devices which are also controlled in other ways or often goes out of sync with Switcher Breeze there are `Assume on` & `Assume off` buttons which you can use to can tell Switcher Breeze if your device is currently running or not without sending a new command to the device.
+
+- For devices which does not support swing status for the vertical swing, the integration provides separate `Vertical swing on` & `Vertical swing off` buttons.
```
Comment (frenck) [inline source/_integrations/switcher_kis.markdown]:
```suggestion
- For devices which do not support swing status for the vertical swing, the integration provides separate `Vertical swing on` & `Vertical swing off` buttons.
```

File: source/_integrations/switcher_kis.markdown [hunk 2]
```diff
@@ -37,17 +37,30 @@ If you completed the integration setup but are still unable to control the devic
 
 {% include integrations/config_flow.md %}
 
+## Buttons
+
+For Switcher Breeze the integration provides the following buttons:
+
+- For devices which are also controlled in other ways or often goes out of sync with Switcher Breeze there are `Assume on` & `Assume off` buttons which you can use to can tell Switcher Breeze if your device is currently running or not without sending a new command to the device.
```
Comment (frenck) [inline source/_integrations/switcher_kis.markdown]:
```suggestion
- For devices that are also controlled in other ways or often go out of sync with Switcher Breeze, there are `Assume on` & `Assume off` buttons which you can use to can tell Switcher Breeze if your device is currently running or not without sending a new command to the device.
```

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by thecode** [comment]
Thanks!

**Pull request merged**